redis/src/client.rs

```diff
@@ -592,6 +592,35 @@ impl Client {
         crate::aio::ConnectionManager::new_with_config(self.clone(), config).await
     }
 
+    /// Returns an async [`ConnectionManager`][connection-manager] from the client.
```

Comment:
this documentation doesn't explain at all the difference of this initializer from the other one. The lazyness needs to be properly explained here.